### DISCUSSION TITLE
Better xcloud 6.0.1

### DISCUSSION THREAD

**Discussion by Ysnow0** [redphx/better-xcloud#586] [General]
Application closes during game login loading screen

**Comment by davinascimentoR** [comment]
tenho uma ideia para colocar no better xcloud: colocar uma opção de adicionar uma conta a mais no app, pra quando for querer jogar com a família conseguir joga tela dividida no jogo, tanto jogo de carro quanto jogo de tiro, a conta adicional que a pessoa for cadastra, isso eu digo em aparelhos; PC, ANDROID, TV E TV BOX.

no jogo call of duty black ops 6, pra joga tela dividida tem que ter uma conta adicional, porem no app ainda não tem essa opção pra jogar assim e nem adicionar uma conta.

tenho certeza que isso iria ajudar muita pessoas a se divertirem com a família e iria agregar bastante.

alias a xcloud esta se expandido cada vez mais.

caso minha ideia seja boa podem tentar entra em contato por e-mail: davigamernr01@gmail.com

**Comment by Ysnow0** [comment]
New Better XCloud 0.22.1-32 version closing during game startup
